webapp/graphite/util.py

```diff
@@ -191,10 +196,15 @@ def load(cls, file):
   unpickle = SafeUnpickler
 
 else:
+  if sys.version_info[0] >= 3:
```

Comment:
This whole branch can be eliminated, every supported python has cPickle

Reply:
As far as I can work out, overriding the Python 3 pickle interface works this way even when it's using the compiled implementation, so we do need both branches. I'll make it clearer what's going on.